resources/views/user/usernewdataset.blade.php

```diff
@@ -104,6 +107,12 @@
                     <span tabindex="0" data-html="true" data-animation="true" class="glyphicon glyphicon-question-sign" data-toggle="tooltip" data-placement="right"
                         title="The usage licence that applies to this layer. Open data is often under a <a href='https://creativecommons.org/licenses/' target='_blank'>Creative Commons</a> CC BY or CC BY-NC licence. If you created the information, you can choose the licence. If you obtained it from another source, select the licence specified there."></span>
                         <input type="text" class="mb-2 w3-white form-control" name="license"/>
+
+                    Image
+                    <span tabindex="0" data-html="true" data-animation="true" class="glyphicon glyphicon-question-sign" data-toggle="tooltip" data-placement="right"
+                      title="Max upload size 4MB">

```

Comment:
Please use the size from configuration in the tooltips.

Reply:
Updated